pom.xml

```
@@ -63,9 +63,11 @@
   </scm>
 
   <properties>
-    <jenkins.version>2.60.3</jenkins.version>
+    <revision>3.1.2</revision>
```

Copy-paste? Use the Maven tool.

```suggestion
    <revision>1.21</revision>
```